src/main/java/appeng/parts/misc/PartStorageBus.java

```diff
@@ -378,10 +378,10 @@ private void resetCache()
 
 		final IMEInventory<IAEItemStack> out = this.getInternalHandler();
 
-		if( this.monitor != null )
-		{
-			this.monitor.onTick();
-		}
+//		if( this.monitor != null )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)